### ISSUE TITLE
Low framerate

### ISSUE THREAD

**Issue by C-Benz** [mierau/gamekid#9]
This is amazing and is the perfect idea for the Playdate. Is this still being worked on and are there any plans to address performance as it runs at what seems like half(ish) speed. Great work anyway!!

**Comment by lunarcloud** [comment]
Maybe a frame-skip option - to avoid drawing but maintain input/sound speed?

**Comment by loonglade** [comment]
> Maybe a frame-skip option - to avoid drawing but maintain input/sound speed?

I agree with this. If games could be locked to 30 fps, I bet it would probably take inputs more smoothly.
